app/services/force/short_form_service.rb

```diff
@@ -120,6 +120,25 @@ def self.submitted?(application)
       application['status'] == 'Submitted'
     end
 
+    def self.lending_institutions
+      endpoint = '/services/apexrest/agents/'
+      institutions = {}
+
+      Request.new.cached_get(endpoint).each do |institution|
+        next unless institution['Contacts'] && institution['Contacts']['records']
+        agents = institution['Contacts']['records'].each_with_object([]) do |array, agent|
+          if agent['BMR_Certified__c']
+            status = agent['Lending_Agent_Status__c'].present? &&
```

Comment:
Can you clarify to me what you're doing with the status here and what the goal is?